src/components/TriPresence.jsx

```diff
@@ -34,7 +34,7 @@ const TriPresence = ({ tri, alt, className, period, ...props }) => {
     if (period === AFTERNOON_PERIOD) {
       return (
         <Avatar className={classes.avatar}>
-          <img src="/public/afternoon.svg" size="16x16" alt="afternoon icon" className={classes.svg} />
+          <img src="afternoon.svg" size="16x16" alt="afternoon icon" className={classes.svg} />
```

Comment:
Par cohérence avec l'icône du matin, je suppose qu'il faut un "/" en préfixe : 

```suggestion
          <img src="/afternoon.svg" size="16x16" alt="afternoon icon" className={classes.svg} />
```